### ISSUE TITLE
[LGTM] Code analysis: Disposable 'HttpClient' is created here but is not disposed.

### ISSUE THREAD

**Issue by jthelin** [jthelin/HelloOwin#34]
LGTM code analysis tools reported the following issue:

**Disposable 'HttpClient' is created here but is not disposed.**

https://github.com/jthelin/HelloOwin/blob/ca2c57dc5fd62164fb58260e828ec07375d53588/HelloOwinClient/HelloOwinClient.cs#L64

Methods that create objects of type 'IDisposable' should call 'Dispose' on those objects, otherwise unmanaged resources may not be released.

https://lgtm.com/projects/g/jthelin/HelloOwin/snapshot/9c5ce827ac773915b0a7a8c0b62ea4216dfb6da5/files/HelloOwinClient/HelloOwinClient.cs?sort=name&dir=ASC&mode=heatmap#x4afce1e98b7fa772:1

**Comment by jthelin** [comment]
SARIF format description of the issue:

```
{
    "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
    "version": "2.1.0",
    "runs": [
        {
            "tool": {
                "driver": {
                    "name": "LGTM.com",
                    "productSuite": "Semmle",
                    "version": "1.23.0-SNAPSHOT",
                    "rules": [
                        {
                            "id": "com.lgtm/csharp-queries:cs/local-not-disposed",
                            "name": "com.lgtm/csharp-queries:cs/local-not-disposed",
                            "shortDescription": {
                                "text": "Missing Dispose call on local IDisposable"
                            },
                            "fullDescription": {
                                "text": "Methods that create objects of type 'IDisposable' should call 'Dispose' on those objects, otherwise unmanaged resources may not be released."
                            },
                            "defaultConfiguration": {},
                            "properties": {
                                "tags": [
                                    "efficiency",
                                    "maintainability",
                                    "external/cwe/cwe-404",
                                    "external/cwe/cwe-459",
                                    "external/cwe/cwe-460"
                                ],
                                "kind": "problem",
                                "precision": "high",
                                "problem.severity": "warning"
                            }
                        }
                    ]
                }
            },
            "artifacts": [
                {
                    "location": {
                        "uri": "HelloOwinClient/HelloOwinClient.cs",
                        "uriBaseId": "%SRCROOT%",
                        "index": 0
                    }
                }
            ],
            "results": [
                {
                    "ruleId": "com.lgtm/csharp-queries:cs/local-not-disposed",
                    "ruleIndex": 0,
                    "message": {
                        "text": "Disposable 'HttpClient' is created here but is not disposed."
                    },
                    "locations": [
                        {
                            "physicalLocation": {
                                "artifactLocation": {
                                    "uri": "HelloOwinClient/HelloOwinClient.cs",
                                    "uriBaseId": "%SRCROOT%",
                                    "index": 0
                                },
                                "region": {
                                    "startLine": 64,
                                    "startColumn": 41,
                                    "endLine": 64,
                                    "endColumn": 57,
                                    "charOffset": 1707,
                                    "charLength": 16
                                }
                            }
                        }
                    ],
                    "partialFingerprints": {
                        "primaryLocationLineHash": "4afce1e98b7fa772:1",
                        "primaryLocationStartColumnFingerprint": "24"
                    }
                }
            ],
            "columnKind": "utf16CodeUnits",
            "properties": {
                "semmle.formatSpecifier": "2.1.0",
                "semmle.sourceLanguage": "csharp"
            }
        }
    ]
}
```

**Comment by jthelin** [comment]
Fixed by PR #1
